HousingRepairsOnlineApi/Controllers/RepairController.cs

```diff
@@ -18,8 +18,8 @@ public class RepairController : ControllerBase
         private readonly IInternalEmailSender internalEmailSender;
         private readonly IRetrieveRepairsUseCase retrieveRepairsUseCase;
         private readonly IRetrieveAvailableCommunalAppointmentUseCase retrieveAvailableCommunalAppointmentUseCase;
-        private readonly IRepairBookingResponseHelper repairBookingResponseHelper;
-
+        private readonly IRepairToRepairBookingResponseMapper _repairToRepairBookingResponseMapper;
```

Comment:
Nitpick: Change name to remove leading underscore
```suggestion
        private readonly IRepairToRepairBookingResponseMapper repairToRepairBookingResponseMapper;
```

Reply:
You may have been looking at an older version - I fixed this naming in the last commit.